project_resource_calendar/models/calendar_event.py

```diff
@@ -179,3 +185,13 @@ def print_calendar_report(self):
         return self.env.ref(
             'project_resource_calendar.calendar_event_report'
         ).report_action(self)
+
+    @api.model
+    def create(self, vals):
```

Comment:
@eilst what if we modify (we use write method) and we delete the client from the participants list? we will not do this verification(verify_client_in_participants)?

Reply:
Dona, right, the validation should also be done when writing